wolfcrypt/benchmark/benchmark.c

```diff
@@ -582,6 +588,9 @@ void bench_rng(void)
 
     END_INTEL_CYCLES
     total = current_time(0) - start;
+#ifdef FREESCALE_KSDK_BM
+    total = total / 1000000;
```

Comment:
Why can't we return seconds (as double) from current_time? It would be nice to not have this logic in every benchmark total, but in the KSDK implementation of current_time.